compiler/resolution/postFold.cpp

```diff
@@ -712,15 +712,15 @@ bool requiresImplicitDestroy(CallExpr* call) {
   bool retval = false;
 
   if (FnSymbol* fn = call->resolvedFunction()) {
-    FnSymbol* parent = call->getFunction();
+    //FnSymbol* parent = call->getFunction();
 
-    if (parent->hasFlag(FLAG_DONOR_FN)                        == false &&
+    if (//parent->hasFlag(FLAG_DONOR_FN)                        == false &&
         isRecord(fn->retType)                                 == true  &&
         fn->hasFlag(FLAG_NO_IMPLICIT_COPY)                    == false &&
         fn->isIterator()                                      == false &&
         fn->retType->symbol->hasFlag(FLAG_RUNTIME_TYPE_VALUE) == false &&
-        fn->hasFlag(FLAG_DONOR_FN)                            == false &&
-        fn->hasFlag(FLAG_INIT_COPY_FN)                        == false &&
+        //fn->hasFlag(FLAG_DONOR_FN)                            == false &&
+        //fn->hasFlag(FLAG_INIT_COPY_FN)                        == false &&
```

Comment:
I suspect you already plan to remove these, just wanted to double-check.

Reply:
yes in fact I plan to entirely remove FLAG_DONOR_FN also. Thanks for the reminder - I had forgotten.